### PR TITLE
Updating README to have the correct URL to releases plus some cargo clippy and fmt fixes

### PR DESCRIPTION
**Pull request by DanielThurau**

### CODE REVIEW

File: src/commands/send.rs [hunk 1]
```diff
@@ -132,19 +132,19 @@ async fn send(message: &Ingress, opts: &SendOpts) -> AnyhowResult {
 }
 
 enum SupportedResponse {
-    ManageNeuronResponse,
-    TransferResponse,
-    AccountBalanceResponse,
+    ManageNeuron,
+    Transfer,
+    AccountBalance,
```
Comment (daniel-wong-dfinity-org) [inline src/commands/send.rs:137]: I guess this is unrelated to the issues mentioned in the title. Also, I guess this does not affect any externally visible interfaces.

**Review by daniel-wong-dfinity-org — APPROVED** [review]

File: src/commands/send.rs (hunk 1 above)
Comment (DanielThurau) [reply src/commands/send.rs:137]: Updated the title right after i sent out the PR. Refresh should have the updated scope. I had to change this to get cargo clippy and cargo fmt to pass
